tests/unit/test-cmd/test.run.js

```diff
@@ -449,6 +431,90 @@ describe('run', () => {
       assert.deepEqual(receivedArgs.ignoreFiles, sentArgs.ignoreFiles);
     });
 
+    it('can reload when user presses R in shell console', () => {
+      const {addonReload, stdin, reloadStrategy} = prepare();
+
+      return reloadStrategy()
+        .then(() => {
+          stdin.emit('keypress', 'r', {name: 'r', ctrl: false});
+        })
+        .then(() => {
+          assert.ok(stdin.setRawMode.called);
+          assert.ok(addonReload.called);
+          assert.equal(addonReload.firstCall.args[0].addonId,
+            tempInstallResult.addon.id);
+        });
+    });
+
```

Comment:
nit: we usually only separate functions by one space (but I guess we don't have an eslint rule for it)